src/metabase/search/ingestion.clj

```diff
@@ -32,6 +33,22 @@
   "The name of the listener that consumes the queue"
   "search-index-update")
 
+(def max-searchable-value-length
+  "The maximum length of a searchable value. This is mostly driven by postgresql max-lengths on tsvector columns.
+  And is about half of postgresql's max, since we concat two values together often. That is likely aggressive, but being safe until we can better understand normal data shapes"
+  500000)
```

Comment:
this is ~0.5mb of text. This feels like its inviting performance issues. I have seen 55mb dataset queries before. I'm not sure anything needs to be done here but this kinda feels wild